recipes/gdal/all/conanfile.py

```diff
@@ -0,0 +1,655 @@
+import glob
+import os
+
+from conans import ConanFile, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+class GdalConan(ConanFile):
+    name = "gdal"
+    description = "GDAL is an open source X/MIT licensed translator library " \
+                  "for raster and vector geospatial data formats."
+    license = "MIT"
+    topics = ("conan", "gdal", "osgeo", "geospatial", "raster", "vector")
+    homepage = "https://github.com/OSGeo/gdal"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "patches/**"
+    generators = "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "simd_intrinsics": [None, "sse", "ssse3", "avx"],
+        "threadsafe": [True, False],
+        "with_zlib": [True, False],
+        "with_libiconv": [True, False],
+        "with_zstd": [True, False],
+        "with_pg": [True, False],
+        # "with_libgrass": [True, False],
```

Comment:
What about these commented options?

Reply:
They require dependencies which are not available yet in CCI.